app/api/sponsors.py

```diff
@@ -41,13 +43,24 @@ class Meta:
                          type_='event')
 
 
-class SponsorList(ResourceList):
+class SponsorListPost(ResourceList):
     """
     List and create Sponsors
     """
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+
+    schema = SponsorSchema
```

Comment:
Nope, we need the `/v1/sponsors` endpoint for GET also, and `/sponsors` uses the SponsorListPost class